lib/msal-common/src/index.ts

```diff
@@ -7,8 +7,8 @@ export { AuthOptions, SystemOptions, LoggerOptions, DEFAULT_SYSTEM_OPTIONS } fro
 export { ClientConfiguration } from "./config/ClientConfiguration";
 // Account
 export { AccountInfo } from "./account/AccountInfo";
-export { IdToken } from "./account/IdToken";
-export { IdTokenClaims } from "./account/IdTokenClaims";
+export { JwtToken } from "./account/JwtToken";
+export { TokenClaims } from "./account/TokenClaims";
```

Comment:
This is a breaking change for common, and thus we would bump the major version. Not sure if a name change is worth a major version bump - should perhaps be discussed as a team.

Reply:
msal-common is an internal package, and thus I don't think it really matters whether or not this is bumped as a major version. Happy to discuss this as a team but this shouldn't be a blocker.

Reply:
Yes, but beacuse its a dependency, new versions can be picked up in customer apps, so we should absolutely avoid breaking changes, in my opinion.

This can be addressed by exporting both:

```suggestion
export { JwtToken } from "./account/JwtToken";
export { JwtToken as IdToken } from "./account/JwtToken";
export { TokenClaims } from "./account/TokenClaims";
export { TokenClaims as IdTokenClaims  } from "./account/TokenClaims";
```

Reply:
done.